library/.releaserc.json

```diff
@@ -17,6 +17,12 @@
         "assets": ["package.json", "CHANGELOG.md"],
         "message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
       }
+    ],
+    [
+      "@semantic-release/exec",
+      {
+        "verifyReleaseCmd": "echo VERSION=${nextRelease.version} >> .env"
```

Comment:
Does this play well with the `writelibvars.sh` script?

Reply:
Yeah works fine, `>>` writes stuff at the end of the file.
The script does the same, order doesn't matter.